test/test.media.ts

```diff
@@ -72,28 +71,13 @@ describe('Media', () => {
   let localDrive, remoteDrive;
   let localGmail, remoteGmail;
 
-  before((done) => {
+  before(async () => {
     nock.cleanAll();
     const google = new GoogleApis();
     nock.enableNetConnect();
-    async.parallel(
-        [
-          (cb) => {
-            Utils.loadApi(google, 'drive', 'v2', {}, cb);
-          },
-          (cb) => {
-            Utils.loadApi(google, 'gmail', 'v1', {}, cb);
-          }
-        ],
-        (err, apis) => {
-          if (err) {
-            return done(err);
-          }
-          remoteDrive = apis[0];
-          remoteGmail = apis[1];
-          nock.disableNetConnect();
-          done();
-        });
+    remoteDrive = await Utils.loadApi(google, 'drive', 'v2');
```

Comment:
I'm just excited how async/await makes the code so much easier.

Reply:
(but probably a little bit slower as we're not parallel here anymore)

Reply:
Thanks!  The serial thing was an accident.  Submitting a fix for that now :)